index.js

```diff
@@ -130,7 +130,7 @@ function parseRubyEngineAndVersion(rubyVersion) {
   } else if (rubyVersion === '.tool-versions') { // Read from .tool-versions
     const toolVersions = fs.readFileSync('.tool-versions', 'utf8').trim()
     const rubyLine = toolVersions.split(/\r?\n/).filter(e => /^ruby\s/.test(e))[0]
-    rubyVersion = rubyLine.match(/^ruby\s+(.+)$/)[1]
+    rubyVersion = rubyLine.split(/\s/)[1]
```

Comment:
```suggestion
    rubyVersion = rubyLine.match(/^ruby\s+(\S+)/)[1]
```
would be another way but what you did seems fine too